src/chrome/content/rules/Borsen.dk.xml

```diff
@@ -1,31 +1,35 @@
 <!--
-	Problematic domains:
+  Problematic domains:
 
-		- ^ ¹
-		- annonceteknik ²
-		- challenge ²
-		- executiveclub ²
-		- finans ¹
-		- gazelle ²
-		- golf ⁴
-		- investor ¹
-		- ledelse ²
-		- play ²
-		- top1000 ³
-		- www ¹
+    - ^ ¹
+    - annonceteknik ²
+    - challenge ²
+    - executiveclub ²
+    - finans ¹
+    - gazelle ²
+    - golf ⁴
+    - investor ¹
+    - ledelse ²
+    - play ²
+    - top1000 ³
+    - www ¹
 
-	¹: Works but serves a lot of strict http content
-	²: 403
-	³: connection timeout
-	⁴: refused
+  ¹: Works but serves a lot of strict http content
+  ²: 403
+  ³: connection timeout
+  ⁴: refused
 -->
 <ruleset name="Borsen.dk (partial)">
 
-	<target host="annonce.borsen.dk" />
-	<target host="img.borsen.dk" />
-	<target host="pleasure.borsen.dk" />
+    <target host="borsen.dk" />
```

Comment:
Please also add `www.borsen.dk` and update the section of `Problematic domains`.